.github/workflows/benchmark.yml

```diff
@@ -154,8 +169,12 @@ jobs:
           chmod +x bombardier
           chmod +x lagging_server
           chmod +x sozu
+          chmod +x bench_logger
       
       - name: ⚡ Launch bench
         working-directory: .github/workflows
-        run:
-          python bench.py
\ No newline at end of file
+        run: python bench.py
+
+      - name: Bench the logs
+        working-directory: .github/workflows
+        run: python bench_logs.py
```

Comment:
The log bench step could have its own job. This will improve the pipline duration by parallelizing jobs

Reply:
done. Seems to work!